lib/vquic/vquic.c

```diff
@@ -410,8 +413,10 @@ static CURLcode recvmsg_packets(struct Curl_cfilter *cf,
         result = CURLE_COULDNT_CONNECT;
         goto out;
       }
-      failf(data, "QUIC: recvmsg() unexpectedly returned %zd (errno=%d)",
-                  nread, SOCKERRNO);
+      char errstr[STRERROR_LEN];
+      Curl_strerror(SOCKERRNO, errstr, sizeof(errstr));
+      failf(data, "QUIC: recvmsg() unexpectedly returned %zd (errno=%d; %s)",
+                  nread, SOCKERRNO, errstr);
```

Comment:
~~~
vquic/vquic.c:416:7: error: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
  416 |       char errstr[STRERROR_LEN];
      |       ^~~~

Reply:
yeah, that failure is unrelated